magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationDelegate.kt

```diff
@@ -57,7 +53,6 @@ public class NavigationDelegate(
   }
 
   override fun onDestroy(context: Context) {
-    backStack.navigables().forEach { removeFromLifecycle(it) }
```

Comment:
we already remove these navigables before this step so this actually crashes when we do it again.